components/ProgressView/tests/snapshot/MDCProgressViewCornerRadiusSnapshotTests.m

```diff
@@ -30,7 +30,7 @@ - (void)setUp {
   //  self.recordMode = YES;
 
   self.progressView = [[MDCProgressView alloc] initWithFrame:CGRectMake(0, 0, 100, 10)];
-  self.progressView.layer.cornerRadius = CGRectGetHeight(self.progressView.bounds) / 2;
+  self.progressView.cornerRadius = CGRectGetHeight(self.progressView.bounds) / 2;
```

Comment:
Please place the rounded fill in explicit tests since the API default is 0.

By default there are no rounded indicators in Material, so we should probably not default to such.

![Screen Shot 2019-05-20 at 10 02 05 AM](https://user-images.githubusercontent.com/1753199/58027334-6362d900-7ae6-11e9-91ee-757681ee248c.png)

Reply:
Sorry! I just saw that the snapshot tests file was not the original one, but is for corner radius testing.